File: 백준/Bronze/29731. 2033년 밈 투표/2033년 밈 투표.py
arr = ["Never gonna give you up",
        "Never gonna stop",
        "Never gonna tell a lie and hurt you",
        "Never gonna say goodbye",
        "Never gonna make you cry",
        "Never gonna run around and desert you",
        "Never gonna let you down"]

count = 0
A = int(input())

for _ in range(A):
    name = input()
    for j in range(len(arr)):
        if(name == arr[j]): count += 1
    
if(A == count): print("No")
else: print("Yes")

File: 백준/Bronze/29731. 2033년 밈 투표/2033년 밈 투표.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    	Scanner scan = new Scanner(System.in);
    	
    	String [] arr = {"Never gonna give you up",
    					"Never gonna stop",
    					"Never gonna tell a lie and hurt you",
    					"Never gonna say goodbye",
    					"Never gonna make you cry",
    					"Never gonna run around and desert you",
    					"Never gonna let you down"};
    	int count = 0;
    	int N = scan.nextInt();
    	
    	for(int i=0; i<N+1; i++) {
    		String name = scan.nextLine();
    		for(int j=0; j<arr.length; j++) {
    			if(name.equals(arr[j])) count++;
    		}
    	}
    	if(N == count) System.out.println("No");
    	else System.out.println("Yes");
    }
}
